src/compat/types_type_kind_ge_413.ml

```diff
@@ -0,0 +1,11 @@
+type ('lbl, 'cstr) type_kind_407 = 
+  | Type_abstract
+  | Type_record of 'lbl list * Types.record_representation
+  | Type_variant of 'cstr list
+  | Type_open
+
+let migrate_type_kind : ('lbl, 'cstr) Types.type_kind -> ('lbl, 'cstr) type_kind_407 = function
+  | Type_abstract -> Type_abstract
+  | Type_record (lbl, repr) -> Type_record (lbl, repr)
+  | Type_variant (cstr, _) -> Type_variant cstr
```

Comment:
`Type_variant` expects two arguments in OCaml 4.13: https://github.com/ocaml/ocaml/commit/1820718aa10139911f07a09d8a2f7783a5aca2a0#diff-4406aa46680eeb3e7bd0ce63737d043ceb84e24f66405d6e9d1dd44d34d38af4R251